multisig/simulations/vip-007/vip-007-sepolia/simulations.ts

```diff
@@ -0,0 +1,147 @@
+    feed: "0x51597f405303c4377e36123cbc172b13269ea163",
+    marketAddress: "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8",
+  },
+  {
+    name: "vUSDT",
+    assetAddress: "0x55d398326f99059fF775485246999027B3197955",
+    feed: "0xb97ad0e74fa7d920791e90258a6e2085088b4320",
+    marketAddress: "0xfD5840Cd36d94D7229439859C0112a4185BC0255",
+  },
+  {
+    name: "vETH",
+    assetAddress: "0x2170Ed0880ac9A755fd29B2688956BD959F933F8",
+    feed: "0x9ef1b8c0e4f7dc8bf5719ea496883dc6401d5b2e",
+    marketAddress: "0xf508fCD89b8bd15579dc79A6827cB4686A3592c8",
+  },
+  {
+    name: "vBTC",
+    assetAddress: "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c",
+    feed: "0x264990fbd0a4796a3e3d8e37c4d5f87a3aca5ebf",
+    marketAddress: "0x882C173bC7Ff3b7786CA16dfeD3DFFfb9Ee7847B",
+  },
+  {
+    name: "vXVS",
+    assetAddress: "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63",
+    feed: "0xbf63f430a79d4036a5900c19818aff1fa710f206",
+    marketAddress: "0x151B1e2635A717bcDc836ECd6FbB62B674FE3E1D",
+  },
+];
+
+forking(5007188, () => {
+  describe("Pre-VIP behavior", () => {
+    let prime: Contract;
+    let primeLiquidityProvider: Contract;
+
+    before(async () => {
+      impersonateAccount(USER);
+      prime = await ethers.getContractAt(PRIME_ABI, PRIME);
+      primeLiquidityProvider = await ethers.getContractAt(PRIME_LIQUIDITY_PROVIDER_ABI, PRIME_LIQUIDITY_PROVIDER);
+    });
+
+    it("prime markets", async () => {
+      expect((await prime.getAllMarkets()).length).to.equal(0);
+    });
+
+    it("prime address", async () => {
+      expect(await primeLiquidityProvider.prime()).to.equal("0x0000000000000000000000000000000000000000");
+    });
+
+    it("claim prime token", async () => {
+      await expect(prime.claim()).to.be.reverted;
+    });
+
+    it("is paused", async () => {
+      expect(await prime.paused()).to.be.equal(true);
+    });
+
+    describe("generic tests", async () => {
+      checkXVSVault();
+    });
+  });
+
+  describe("Post-VIP behavior", async () => {
+    let prime: Contract;
+    let primeLiquidityProvider: Contract;
+    let xvsVault: Contract;
+    let xvs: Contract;
+
+    before(async () => {
+      await pretendExecutingVip(vip007());
+
+      impersonateAccount(USER);
+      const accounts = await ethers.getSigners();
+      await accounts[0].sendTransaction({ to: USER, value: parseUnits("10") });
+
+      const signer = await ethers.getSigner(USER);
+      prime = await ethers.getContractAt(PRIME_ABI, PRIME, signer);
+      primeLiquidityProvider = await ethers.getContractAt(PRIME_LIQUIDITY_PROVIDER_ABI, PRIME_LIQUIDITY_PROVIDER);
+      xvsVault = await ethers.getContractAt(XVS_VAULT_ABI, XVS_VAULT_PROXY, signer);
+      xvs = await ethers.getContractAt(ERC20_ABI, XVS, signer);
+
+      for (let i = 0; i < vTokens.length; i++) {
+        const vToken = vTokens[i];
+        await setMaxStalePeriodInChainlinkOracle(CHAINLINK_ORACLE, vToken.assetAddress, vToken.feed, GUARDIAN);
```

Comment:
`setMaxStalePeriod` is a simplified version of this, that I think we could use. It's up to you